java/code/src/com/redhat/rhn/manager/content/MgrSyncUtils.java

```diff
@@ -200,6 +200,17 @@ public static ProductName findOrCreateProductName(String name) {
         return productName;
     }
 
+    /**
+     * Converts the specified network url to a file system url
+     * @param urlString the url
+     * @param name the name of the repo
+     * @return the file system URI
+     */
+    public static URI urlToFSPath(String urlString, String name) {
+        String sccDataPath = Config.get().getString(ContentSyncManager.RESOURCE_PATH, null);
```

Comment:
ConfigDefaults got a new method "getOfflineMirrorDir()". Why is this not used here?